apps/meteor/app/statistics/server/lib/getContactVerificationStatistics.ts

```diff
@@ -0,0 +1,44 @@
+import type { IStats } from '@rocket.chat/core-typings';
+import { LivechatContacts } from '@rocket.chat/models';
+
+import { settings } from '../../../settings/server';
+
+export async function getContactVerificationStatistics(): Promise<IStats['contactVerification']> {
+	const [
+		totalContacts,
+		totalUnknownContacts,
+		[{ totalConflicts, avgChannelsPerContact } = { totalConflicts: 0, avgChannelsPerContact: 0 }],
+		totalBlockedContacts,
+		totalFullyBlockedContacts,
+		totalVerifiedContacts,
+		totalContactsWithoutChannels,
+	] = await Promise.all([
+		LivechatContacts.estimatedDocumentCount(),
+		LivechatContacts.countDocuments({ unknown: true }),
```

Comment:
pls move these queries to the model.. i.e.:

- `LivechatContacts.countUnknown()`
- `LivechatContacts.countKnown()`
- `LivechatContacts.countChannelsBlocked()`
- ...

Reply:
Done!